internal/model/db.go

```diff
@@ -0,0 +1,12 @@
+package model
+
+import (
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+)
+
+func ConnectDatabase(databseUrl string) (db *gorm.DB, err error) {
```

Comment:
Where is Close Database function?)



Database should be closed after usage

Reply:
🤔 in documentation i don't see what in gorm needed close connection

i leave it as is before we meet connection pool error)

.Close() function i saw used in simple sql connection